### ISSUE TITLE
Define standard annotation language for M2 documentation

### ISSUE THREAD

**Issue by enikao** [LionWeb-io/specification#290]
In https://github.com/LionWeb-io/lionweb-mps/pull/94, we've added annotations on exported language elements to keep MPS' _shortDescription_ and _conceptAlias_ in the export. We're using this in https://github.com/LionWeb-io/lionweb-csharp/pull/13.

However, this is a very MPS-specific solution. A platform-agnostic language to store such annotations would be better.

**Comment by dslmeinte** [comment]
Would it make sense to add these annotations to the `Builtins` language?

**Comment by enikao** [comment]
I don't think so: `Builtins` is for stuff the system absolutely doesn't work without -- I'd like to keep it as small as possible. These kind of descriptions are useful, but not necessary at all. An implementation can perfectly work without supporting descriptions, I would not like to force support.
